### DISCUSSION TITLE
JsonLd set hiringOrganization as confidential but only shows the string confidential expected?

### DISCUSSION THREAD

**Discussion by 28development** [schemaorg/schemaorg#3169] [Q&A]
Hi,

Not sure if this is the expected behavior so I am just asking here:

We added JsonLd to our site and it works good so far, but I think that the confidential property does not really work in our setup.

This is out setup:

```
{
   "@context":"https://schema.org",
   "@type":"JobPosting",
   "datePosted":"2022-08-25",
   "description":"exam. Altenpflegefachkraft (m/w/d) - Vielfalt am Arbeitsplatz entdecken!",
   "title":"exam. Altenpflegefachkraft (m/w/d) - Vielfalt am Arbeitsplatz entdecken!",
   "employmentType":[
      "full_time",
      "part_time"
   ],
   "baseSalary":{
      "@type":"MonetaryAmount",
      "currency":"EUR",
      "value":{
         "@type":"QuantitativeValue",
         "unitText":"MONTH",
         "minValue":3100,
         "maxValue":3600
      }
   },
   "hiringOrganization":"confidential",
   "jobLocation":{
      "@type":"Place",
      "address":{
         "@type":"PostalAddress",
         "addressCountry":"Deutschland",
         "addressLocality":"München",
         "postalCode":"81925"
      }
   },
   "applicantLocationRequirements":{
      "@type":"Country",
      "name":"DE"
   }
}
```

This is how we set the confidential prop like it is described in the docs:

`   "hiringOrganization":"confidential",`

But currently the JSON-LD view shows as a company name "confidential" which is wrong I think, what are we missing here?

Kind regards.

**Comment by danbri** [comment]
This looks like a Google issue not a Schema.org thing.

the Schema.org definition doesn’t talk about “Confidential” as a special value for https://schema.org/hiringOrganization

But in https://developers.google.com/search/docs/advanced/structured-data/job-posting

> organization offering the job position. This must be the name of the company (for example, "Starbucks, Inc"), and not the specific location that is hiring (for example, "Starbucks on Main Street"). 
…
> If the organization is hiring anonymously (for example, a staffing agency on behalf of an anonymous employer or an employer directly on your platform), use the confidential value for the HiringOrganization field.

I will investigate at Google and report back.

**Reply by 28development** [reply]
Hi @danbri thanks for the reply, are you going update on this thread once you have more information.

**Comment by danbri** [comment]
I will.

**Reply by 28development** [reply]
Any updates on this one, would be interesting 🙏

**Comment by mountbatt** [comment]
@danbri @28development Any news on this?

**Comment by danbri** [comment]
@RichardWallis - can you take a look?

**Comment by RichardWallis** [comment]
Echoing @danbri's comment:
_This looks like a Google issue not a Schema.org thing.
the Schema.org definition doesn’t talk about “Confidential” as a special value for https://schema.org/hiringOrganization_

Looking at Googles writings on this there is some confusion and lack of clarity.

Firstly they say this [in documentation](https://schema.org/hiringOrganization):
_If the organization is hiring anonymously (for example, a staffing agency on behalf of an anonymous employer or an employer directly on your platform), **use the confidential value** for the HiringOrganization field._
However, there is no clarification as to what the _confidential value_ is - or is it just the string "confidential"?

Elsewhere there is this in a Google support conversation:
_I don't know which documents you are referring to, but this markup complies with [Google's ban](https://developers.google.com/search/docs/advanced/structured-data/job-posting#misrepresentation):_

_Jobs or content that represent the employer in a way that is not accurate, realistic, or truthful._
 
_Also this markup contradicts [Google](https://developers.google.com/search/docs/advanced/structured-data/job-posting#job-posting-definition):_

_hiringOrganization [Organization](https://schema.org/Organization)_
_The organization offering the job position. This **must be the name of the company**_

As a data consumer, some clarity from Google would be helpful.

One option for Schema.org could be to to define a specific value in the vocabulary to represent this state of a confidential organization. However, this would then raise issues of where this would be applicable and which data consumers may adopt it.
